src/modeler/AutoRest.Swagger/Validation/Core/Rule.cs

```diff
@@ -27,6 +27,11 @@ protected Rule()
         public virtual ValidationCategory ValidationCategory => ValidationCategory.None; // !!! implement me and make me abstract !!!
 
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public virtual ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.None;
```

Comment:
I'm thinking links could be fwlinks, so we don't need to update the code if the link changes, I'd like to hear @kirthik thoughts on this as well. If not, it could also be a path to an top level page, that doesn't change frequently and would have a list of all rules with links to each specific one.